textbook/12/1/Probability-Sampling.ipynb

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
<p>I'm wondering if the information in this chapter should be split between other chapters? By this time they have already heard many of the concepts that this chapter addresses, using this chapter to build on them is one option but I'm not sure if the flow would be better if we added this to chapters on randomness and data collection and probability... Does anyone have thougthts?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>